### PR TITLE
Earmark version bump

### PR DESCRIPTION
**Pull request by RobertDober**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 defmodule ExDoc.Mixfile do
   use Mix.Project
 
-  @version "0.21.2"
+  @version "0.19.1"
```
Comment (RobertDober) [inline mix.exs]: I just pulled from upstream, strange????

**Comment by josevalim** [comment]
Hi @RobertDober! Is there any reason for us to enforce the latest? I typically don't like to enforce the latest and give users the control for them to do if they want to. After all, they can simply do `mix deps.update earmark`. What do you think?

**Comment by eksperimental** [comment]
@josevalim it fixes this issue:
* https://github.com/elixir-lang/ex_doc/issues/1108
* https://github.com/pragdave/earmark/issues/309

You can read it in this commit https://github.com/pragdave/earmark/commit/8f533b11dc4a10ba42101fd9514b234e6c64c72a

**Comment by josevalim** [comment]
Yes, but folks can get those changes without us imposing a stricter requirement. I will bump to ~> 1.4 but generally speaking we should not be very strict on the requirements unless we have strong reasons to. Thanks!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by eksperimental** [comment]
Correct!. `~> 1.4.3` wouldn't have included `1.5.0` when this one is released.

**Comment by RobertDober** [comment]
I completely yield to you, the ex_doc team for that issue.
I will only send PRs for minor version bumps in the future.
Makes sense too :)

**Pull request merged**